core/src/test/java/org/elasticsearch/watcher/ResourceWatcherServiceTests.java

```diff
@@ -53,9 +53,9 @@ public void testSettings() throws Exception {
 
         // checking custom
         settings = Settings.builder()
-                .put("resource.reload.interval.high", "10s")
-                .put("resource.reload.interval.medium", "20s")
-                .put("resource.reload.interval.low", "30s")
+                .put(ResourceWatcherService.RELOAD_INTERVAL_HIGH.getKey(), "10s")
```

Comment:
I prefer the string constants in tests, rather than using the key. The reason for this is because it helps catch inadvertent changes to the key. That is, if an inadvertent change is made to the string constant in `ResourceWatcherService.RELOAD_INTERVAL_HIGH` then this test will continue to pass but it would be better if it failed. I've made comments along these lines [previously](https://github.com/elastic/elasticsearch/pull/17778#discussion_r59856263). That's my reasoning, up to you how to process.